src/index.js

```diff
@@ -0,0 +1,155 @@
+};
+
+const section = document.querySelector('.section');
+section.innerHTML = `
+  <div class="main-div">
+    <p id="header">Today's To Do <i class="fas fa-sync"></i></p>
+    <form action="" class="form">
+      <input class="dataEntry" type="text" placeholder="Add to your list..." required></input>
+    </form>
+    <p id="clear">Clear all completed</p>
+  </div>
+`;
+// Create list
+const createList = () => {
+  const form = document.querySelector('.form');
+  const list = document.createElement('div');
+  list.className = 'input-div';
+  form.appendChild(list);
+  const checkboxes = document.createElement('input');
+  checkboxes.className = 'input';
+  checkboxes.type = 'checkbox';
+  const listText = document.createElement('p');
+  listText.className = 'listContent';
+  const threeDots = document.createElement('i');
+  threeDots.className = 'fas fa-ellipsis-v';
+  const trashIcon = document.createElement('i');
+  trashIcon.className = 'fas fa-trash-alt icon2';
+  list.append(checkboxes, listText, threeDots, trashIcon);
+  // Add event to checkboxes
+   let count = 1;
+  checkboxes.addEventListener('click', () => {
+    threeDots.classList.toggle('remove-icon-active');
+    trashIcon.classList.toggle('icon2');
+    listText.classList.toggle('listContent-disable');
+    list.classList.toggle('changeBg');
+    const getting = JSON.parse(localStorage.getItem('list'));
+    const empty = [];
+    const hammasi = document.querySelectorAll('.input-div');
+    for (let i = 0; i < getting.length; i += 1) {
+      if (hammasi[i].classList.contains('changeBg')) {
+        getting[i].completed = true;
+        count += 1;
+      } else {
+        getting[i].completed = false;
+      }
+      empty.push(getting[i]);
+      localStorage.setItem('list', JSON.stringify(empty));
+    }
+  });
+  const clearAll = document.querySelector('#clear');
+  clearAll.addEventListener('click', () => {
+    const getting = JSON.parse(localStorage.getItem('list'));
+    const variable = document.querySelectorAll('.changeBg');
+    for (let i = 0; i < variable.length; i += 1) {
+      form.removeChild(variable[i]);
+    }
+    const empty = [];
+    for (let i = 0; i < getting.length; i += 1) {
+      if (getting[i].completed === true) {
+        continue;
+      }
+      empty.push(getting[i]);
+    }
+    localStorage.setItem('list', JSON.stringify(empty));
+  });
+
+  // Remove from list event
+  trashIcon.addEventListener('click', () => {
+    form.removeChild(list);
+    const getFromLocalStorage = JSON.parse(localStorage.getItem('list'));
+    const result = getFromLocalStorage.filter((word) => word.description === listText.textContent);
+    const empty = [];
+    for (let i = 0; i < getFromLocalStorage.length; i += 1) {
+      if (result[0].description === getFromLocalStorage[i].description) {
+        continue;
+      }
+      empty.push(getFromLocalStorage[i]);
+    }
+    localStorage.setItem('list', JSON.stringify(empty));
+  });
```

Comment:
- [ ] kindly consider converting your code for removing a task to a function so your app has a function to removing a task

Reply:
- Kindly see to this change it has not been implemented yet.